### ISSUE TITLE
The mix data set cannot achieve the expected effect

### ISSUE THREAD

**Issue by yifengW** [ifzhang/FairMOT#273]
Thank you for your contribution.When I use mot17 for training and mot17 for testing, the expected results can be achieved. However, when I use the mix for training and mot17 for testing， the experiment results can not achieved the 70 for MOTA. only 64. I can not know what I need to do to solve the problems.

**Comment by ifzhang** [comment]
You can check whether your mix dataset containing MOT17 train. I use Caltech, Cityperson, ETH, PRW, CUHK-SYSU and MOT17 to train the model and it can achieve 72+ MOTA on MOT17 test set. You can also try to use MIX dataset (5 datasets) and half MOT17 train to train your model and evaluate on another half of MOT17 train to see whether it can achieve 69+ MOTA.

**Comment by yifengW** [comment]
Thank you. I have achieved the results by checking the setting.


| |
王益峰
|
|
yifeng1129@163.com
|
签名由网易邮箱大师定制


On 11/12/2020 14:31，Yifu Zhang<notifications@github.com> wrote：

You can check whether your mix dataset containing MOT17 train. I use Caltech, Cityperson, ETH, PRW, CUHK-SYSU and MOT17 to train the model and it can achieve 72+ MOTA on MOT17 test set. You can also try to use MIX dataset (5 datasets) and half MOT17 train to train your model and evaluate on another half of MOT17 train to see whether it can achieve 69+ MOTA.

**Comment by iamZe** [comment]
@yifengW ,is there any tips for achieving the reported mota on mot17 following the author's instructions? thanks!

**Comment by yifengW** [comment]
No, following the authors’ instructions can achieve the performance. I find that the conference is very important.
| |
王益峰
|
|
yifeng1129@163.com
|
签名由网易邮箱大师定制


On 11/23/2020 14:48，iamZe<notifications@github.com> wrote：

@yifengW ,is there any tips for achieving the reported mota on mot17 following the author's instructions? thanks!

**Comment by iamZe** [comment]
@yifengW are you referring to the confidence score? which should be 0.4 on mot17 right?

**Comment by tahminp** [comment]
Hi, 

Please let me know I can prepare Mix datasets for train and test after downloading. In which format they should be?
